app/javascript/src/lib/OWLanguageLinter.ts

```diff
@@ -1,11 +1,13 @@
 import { findRangesOfStrings, getClosingBracket, getPhraseFromPosition, matchAllOutsideRanges, splitArgumentsString } from "@utils/parse"
-import { completionsMap, subroutinesMap, workshopConstants } from "@stores/editor"
+import { completionsMap, currentItem, modal, subroutinesMap, workshopConstants } from "@stores/editor"
 import { get } from "svelte/store"
 import { getFirstParameterObject } from "@utils/compiler/parameterObjects"
 import type { EditorView } from "codemirror"
-import type { Diagnostic } from "@codemirror/lint"
-import type { Severity } from "@src/types/editor"
+import { type Diagnostic } from "@codemirror/lint"
```

Comment:
Any reason for changing this? The others are all `import type`

Reply:
That's my IDE doing IDE stuff. I think its because I had prettier setup.

Reply:
```suggestion
import type { Diagnostic } from "@codemirror/lint"
```